scripts/ai/AIDriveStrategyShovelSiloLoader.lua

```diff
@@ -435,11 +435,20 @@ function AIDriveStrategyShovelSiloLoader:hasTrailerValidSpecializations(trailer)
         --- All normal trailers
         return true
     end
-    if SpecializationUtil.hasSpecialization(Sprayer, trailer.specializations)
-            and trailer.spec_sprayer.isManureSpreader then
+    if SpecializationUtil.hasSpecialization(Sprayer, trailer.specializations) then
+       --     and trailer.spec_sprayer.isManureSpreader then
```

Comment:
Is this condition intentionally commented out?

Reply:
leftover comment will remove it :)